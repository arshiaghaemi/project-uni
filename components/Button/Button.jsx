import React from "react";
import { css } from "@emotion/css";
import { useTheme } from "@emotion/react";

const Button = ({children, type, onClick}) => {
    const theme = useTheme()
    return(
        <button
        onClick={onClick}
        className = {css`
        padding: 32px;
        border: none;
        color: yellow;
        font-size: 17px;
        background-color:blue;
        margin: 10px;
        `}
    >
        {children}
    </button>
    )
}

export default Button