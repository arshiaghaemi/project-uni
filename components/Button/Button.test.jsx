import Button from "./Button"
import React from "react";
import renderer from 'react-test-renderer';
describe('Button', () => {
    test('Render correctly', () => {
        const component = renderer.create(
            <Button>sample Button</Button>
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})