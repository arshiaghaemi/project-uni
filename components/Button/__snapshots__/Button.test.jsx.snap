// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Render correctly 1`] = `
<button
  className="css-863byk"
>
  sample Button
</button>
`;
